build.fsx

```diff
@@ -322,7 +322,7 @@ Target "QuickIntegrationTests" (fun _ ->
 // --------------------------------------------------------------------------------------
 // Build a NuGet package
 
-let mergeLibs = ["paket.exe"; "Paket.Core.dll"; "FSharp.Core.dll"; "Newtonsoft.Json.dll"; "Argu.dll"; "Chessie.dll"; "Mono.Cecil.dll"]
```

Comment:
No need to ILMerge this?

Reply:
Not for current platforms as it's part of netstandard, but might be needed to support older platforms and/or resolve discussed version conflict/s